src/database/config/buildDb.ts

```diff
@@ -0,0 +1,15 @@
+import { readFileSync } from 'fs';
+import { join } from 'path';
+import connection from './connection';
+
+const buildDb = async () => {
+  const buildFile = readFileSync(join(__dirname, 'build.sql')).toString();
+  const fakeDataFile = readFileSync(join(__dirname, 'fakeData.sql')).toString();
+  try {
+    await connection.query(buildFile + fakeDataFile);
+  } catch (err) {
+    console.log('DB Error: ', err);
+  }
+};

```

Comment:
Why did you create this file? Khaled created it already, delete it please

Reply:
ok, deleted it.